discord/role.py

```diff
@@ -341,7 +341,14 @@ def created_at(self) -> datetime.datetime:
 
     @property
     def mention(self) -> str:
-        """:class:`str`: Returns a string that allows you to mention a role."""
+        """:class:`str`: Returns a string that allows you to mention a role.
+
+        .. versionchanged:: 2.0
```

Comment:
Is this necessary? The original behavior of "Returns a string that allows you to mention a role." hasn't changed.

Reply:
It has changed.
![Screenshot_20220612-184439_Discord.jpg](https://user-images.githubusercontent.com/13399568/173243838-842fbac9-b9ee-4ab9-b434-2824d7e22bda.jpg)

Reply:
No, I meant it still "Returns a string that allows you to mention a role."